src/test/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/prisonersearch/PrisonerSearchResultsProcessorTest.kt

```diff
@@ -35,11 +34,20 @@ class PrisonerSearchResultsProcessorTest {
       "aliasExactMatchCount" to "0",
       "singleResultIdentified" to "true"
     )
+    val expectedBestMatchCustomDimensions = mapOf(
+      "status" to null,
+      "inOutStatus" to null,
+      "legalStatus" to null,
+      "lastMovementTypeCode" to null,
+      "lastMovementReasonCode" to null,
+      "forwardingRequired" to "true",
+      "hasCellLocation" to "false"
```

Comment:
Might be worth running this past Liora at some point to check if there is anything else they want to capture - they mentioned restricted hospital info at one point but not heard since.

Reply:
Yeah, we can easily add to it now that its not that nasty logger class I originally wrote 😁